src/test/java/tests/LogInTest.java

```diff
@@ -15,9 +18,15 @@
     @Test
     public void logInToAcc() {
         homePage.openHomePage();
+        WebElement element = driver.findElement(By.xpath("//button[@class='button button_color_green button_size_medium search-form__submit']"));
```

Comment:
all webelements and driver methods should be used on Page level. We can't use it on Tests level